### ISSUE TITLE
Accessibility regression

### ISSUE THREAD

**Issue by dmpr0** [osmandapp/OsmAnd#9934]
Can't select any POI / Favorites on map
-> First tap on icon and then double tap doesn't open context menu 

-- It seems that in the current release, double-tap doesn't work if the "Accessibility" plugin is enabled.
